src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -26,6 +27,8 @@ public SimulationCache(WorldGenerationSettings worldSettings)
             this.worldSettings = worldSettings;
         }
 
+        public bool DayNightCycleEnabled => !worldSettings.DayNightEnabled;
```

Comment:
Is the bool the wrong way around here?

Reply:
Probably, I hesitated between having enabled and disabled, so it must be a leftover... i'll correct that when I get the chance to.